FinancialTracker/Helpers/Currency.swift

```diff
@@ -101,3 +40,51 @@ extension Currency {
         rate = try values.decodeIfPresent(Double.self, forKey: .rate) ?? 0
     }
 }
+
+func getCurrencies(completionHandler: @escaping (String?, [Currency]?) -> Void) {
+    var allCurrencies: [Currency] = []
```

Comment:
Seems like this property is not needed.

Reply:
The property `allCurrencies` is returned in the completion handler.

Reply:
got it